doc/ipythonconsole.rst

```diff
@@ -2,91 +2,145 @@
-Supported features
-==================
+   You can get :file:`{jupyter/runtime/dir/path}` by executing ``jupyter --runtime-dir`` in the same Python environment as the kernel.
+   Usually, the connection file you are looking for will be one of the newest in this directory, corresponding to the time you started the external kernel.
 
-Any :guilabel:`IPython Console` in Spyder, internally or externally created, supports additional features including:
+#. Click :guilabel:`Connect to an existing kernel` from the :guilabel:`Console` menu or the :guilabel:`IPython Console` pane's "Gear" menu.
 
-.. image:: images/console/console-completion.png
-   :alt: Spyder IPython Console, with a popup list of code completion guesses
+#. Browse for or enter the path to the connection file from the previous step.
 
-* Automatic code completion
-* Real-time function calltips
-* Debugging toolbar integration for launching the debugger and controlling execution flow
+   As a convenience, kernel ID numbers (e.g. ``1234``) entered in the connection file path field will be expanded to :file:`{jupyter/runtime/dir/path}/kernal-{id}.json` on your local machine, if you've copied the connection file there.
 
-Spyder-created consoles support even more advanced capabilities, such as:
+#. Check the :guilabel:`This is a remote kernel (via SSH)` box and type the full hostname you're connecting to (in the form :file:`{username}@{hostname}:{port-number}`).
+   Then, enter *either* :file:`{username}`'s password on the remote machine, or your user SSH keyfile (typically :file:`.perm`) (only one is needed to connect), and press :guilabel:`OK`.
 
-* The :doc:`variableexplorer`, with GUI-based editors for many built-in and third-party Python objects
-* Full GUI integration with the enhanced IPython debugger, ``ipdb``, including viewing and setting normal and conditional breakpoints interactively in any file, a :guilabel:`Breakpoints` pane, and following along with execution flow in the in the :doc:`editor` (see the :doc:`debugging` documentation for more details)
-* The :ref:`User Module Reloader <umr-section>`, which can automatically re-import modified packages and files
-* Inline display of Matplotlib graphics, if the ``Inline`` backend is selected under :menuselection:`Preferences --> IPython console --> Graphics --> Graphics backend`
+   The port number is the one on which the SSH daemon (``sshd``) is running, typically 22 unless you or your administrator has configured it otherwise.
 
-For information on the features, commands and capabilities built into IPython itself, see the `IPython documentation`_.
+(Gif of connecting to remote kernel SSH ->CAM)
+
+For more technical details about connecting to remote IPython kernels, see the `Connecting to a remote kernel`_ page in the IPython Cookbook.
+Just remember to enter the appropriate details into Spyder's :guilabel:`Connect to an existing kernel` dialog instead of launching a new frontend on the client with ``--existing``.
```

Comment:
I don't understand what this sentence is about. Could you rephrase it?

Reply:
Its basically saying to enter the details into Spyder's dialog instead of using IPython as the frontend, but I agree its confusing. As this is an advanced topic (as mentioned in the previous sentence), I think its best to just eliminate this sentence since we already explain it better above.

```suggestion
```